vespa/test_integration_docker.py

```diff
@@ -132,8 +132,12 @@ def create_sequence_classification_task():
 
 
 class TestDockerCommon(unittest.TestCase):
-    def deploy(self, application_package, disk_folder):
-        self.vespa_docker = VespaDocker(port=8089, disk_folder=disk_folder)
+    def deploy(self, application_package, disk_folder, container_image=""):
+        if len(container_image) > 0:
```

Comment:
When using `container_image = None` as per my suggestion above, all that is needed is to use `if container_image:`